JULIERAJ/KIDS-FIRST-Front
# Feat/40/remember me

## Remember me + Release Fixes

## Requirements
- Set up Remember me Option
- Fix bugs in flow for the release

## Test Steps
- Public Routes are set up: when you are logged in you can't have access to sign in/up and landing page
- Error messages in auth flow match Figma 
- If you press remember me on the sign in page, your cookies will expire in a month otherwise they will expire in a day. If you log in with google, the cookies lifetime will be 30 days by default - you shouldn't tap remember me for that.
For testing open developer's tools => application  => cookies => find token and user, see the expiration date and time.
- Sidebar icon has better quality
- Log out button should be over the help button 

## Notes
-  **Check with the backend branch feat/40/remember me**
- Two error messages in auth flow do not match Figma - to be addressed later (to be discussed with QA/design team)

## Checklist
